front/app/community/[siren]/components/FicheMarchesPublics/SectorTable.tsx

```diff
@@ -0,0 +1,91 @@
+'use client';
+
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/components/ui/table';
+import { formatNumber } from '@/utils/utils';
+
+type Sector = {
+  name: string;
+  size: number;
+  part: number;
+  pourcentageCategoryTop1: number;
+};
+
+type TreeLeaf = {
+  type: 'leaf';
+  name: string;
+  value: number;
+};
+
+type TreeNode = {
+  type: 'node';
+  value: number;
+  name: string;
+  children: Tree[];
+};
+
+type Tree = TreeNode | TreeLeaf;
+
+type SectorTableProps = {
+  topSectors: Sector[];
+  formattedData: Tree;
+  linesDisplayed: number;
+  onLoadMore: () => void;
+}
+
+export default function SectorTable({
+  topSectors,
+  formattedData,
+  linesDisplayed,
+  onLoadMore,
+}: SectorTableProps) {
+  return (
+    <>
+      <Table className='min-h-[600px]'>
+        <TableHeader>
+          <TableRow>
+            <TableHead className='w-[400px]'>Secteur</TableHead>
+            <TableHead className='w-[700px]'>Montant</TableHead>
+            <TableHead className=''></TableHead>
+            <TableHead className='text-right'>Part</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {topSectors.map((item, index) => (
+            <TableRow key={index}>
+              <TableCell className='font-medium'>{item.name}</TableCell>
+              <TableCell>
+                <div className='relative h-2 w-full rounded-md'>
+                  <div
+                    className='h-2 rounded-md bg-blue-500'
+                    style={{ width: `${item.pourcentageCategoryTop1}%` }}
+                  ></div>
+                </div>
+              </TableCell>
```

Comment:
Tu peux creer un petit composant `PercentageBarCell` que l on retulisera dans les tables

```suggestion
              <PercentageBarCell value={item.pourcentageCategoryTop1}/>
```

Reply:
Done